public/index.html

```diff
@@ -0,0 +1,85 @@
+<html>
+  <head>
+    <title>Anti-Tourist-Trap</title>
```

Comment:
@Rubiksron You wanted the site to be mobile friendly, but not `<meta>` tag for the viewport?